test/distributed/org/apache/cassandra/fuzz/sai/SingleNodeSAITestBase.java

```diff
@@ -136,17 +164,20 @@ private void basicSaiTest(EntropySource rng, SchemaSpec schema)
         cluster.forEach(i -> i.nodetool("disableautocompaction"));
 
         cluster.schemaChange(schema.compile());
-        cluster.schemaChange(schema.compile().replace(schema.keyspace + "." + schema.table,
-                                                      schema.keyspace + ".debug_table"));
-        Streams.concat(schema.clusteringKeys.stream(),
-                       schema.regularColumns.stream(),
-                       schema.staticColumns.stream())
+        cluster.schemaChange(schema.compile().replace(schema.keyspace + '.' + schema.table, schema.keyspace + ".debug_table"));
+
+        Streams.concat(schema.clusteringKeys.stream(), schema.regularColumns.stream(), schema.staticColumns.stream())
                .forEach(column -> {
+                   if (createIndex.get())
```

Comment:
feel free to ignore, but this might be better as

```
if (createIndex.test(rng, column))
```

you can still do the `(i1, i2) -> true` that you do in most of the tests, but does give you the flexibility for more randomized coverage

Reply:
turns out you do `() -> rng.nextFloat() < 0.7f`.  So this idea isn't too crazy =D